module/RoutePlanner.cpp

```diff
@@ -0,0 +1,16 @@
+/**
+ * @file RoutePlanner.cpp
+ * @brief 経路計画クラス
+ * @author hiroto0927, miyashita64
+ */
+
+#include "RoutePlanner.h"
+
+// コンストラクタ
+RoutePlanner::RoutePlanner() : destinationList(bingoArea), blockSelector(bingoArea, destinationList)
+{
+  // ブロックの運搬順を取得する
+  for(int i = static_cast<int>(BLOCK_ID::ID0); i < static_cast<int>(BLOCK_ID::ID7) + 1; i++) {
+    blockOrder[i] = blockSelector.selectBlock();
+  }
+}
```

Comment:
staticクラスにしたらこんな感じかな

```suggestion
void RoutePlanner::runBlockBingoArea()
{
  std::array<BLOCK_ID, static_cast<int>(BLOCK_ID::ID7) + 1> blockOrder;
  BingoArea bingoArea;
  DestinationList destinationList(bingoArea);
  BlockSelector blockSelector(bingoArea, destinationList);
  
  // ブロックの運搬順を取得する
  for(int i = static_cast<int>(BLOCK_ID::ID0); i < static_cast<int>(BLOCK_ID::ID7) + 1; i++) {
    blockOrder[i] = blockSelector.selectBlock();
  }
}
```